ts/import-csv/DupeResolutionSelector.svelte

```diff
@@ -0,0 +1,56 @@
+<!--
+Copyright: Ankitects Pty Ltd and contributors
+License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
+-->
+<script lang="ts">
+    import Col from "../components/Col.svelte";
+    import Row from "../components/Row.svelte";
+    import * as tr from "../lib/ftl";
+    import { ImportExport } from "../lib/proto";
+
+    const DupeResolution = ImportExport.ImportCsvRequest.DupeResolution;
+
+    export let dupeResolution: ImportExport.ImportCsvRequest.DupeResolution =
+        DupeResolution.IGNORE;
+
+    const dupeResolutions = allResolutions();
+
+    let index = dupeResolutions.findIndex((entry) => entry.value === dupeResolution);
+    $: dupeResolution = dupeResolutions[index].value;
+
+    interface IResolution {
+        value: ImportExport.ImportCsvRequest.DupeResolution;
+        label: string;
+    }
+
+    function allResolutions(): IResolution[] {
+        return [
+            {
+                value: DupeResolution.ADD,
+                label: tr.importingImportEvenIfExistingNoteHas(),
+            },
+            {
+                value: DupeResolution.IGNORE,
+                label: tr.importingIgnoreLinesWhereFirstFieldMatches(),
+            },
+            {
+                value: DupeResolution.UPDATE,
+                label: tr.importingUpdateExistingNotesWhenFirstField(),
+            },
+        ];
+    }
+</script>
+
+<Row --cols={2}>
+    <Col --col-size={1}>
+        {tr.importingDuplicateResolution()}
```

Comment:
Is this understandable? I also think the combobox option labels below are cumbersome. I know they were made for a larger box without label in the old screen. But I couln't come up with something better.

Reply:
How about "Duplicate resolution" -> "Existing notes"

If removing the label and using the entire row for the selection box doesn't look good, maybe we could add some new translations instead, that were shorter, eg "Update", "Preserve", "Duplicate". We could add a tooltip with more detailed explanations if the short forms were not clear enough.

Reply:
Sounds good! I'll leave tooltips out for now, but keep it in mind.